docs/scripting.md

```diff
@@ -493,6 +493,16 @@ Using previous examples:
 
 For the second example, it's worth thinking about what messages the user would see. If you have an error handler that replies to the user, you may not need to add a custom message and could send back the error message provided to the `get()` request, but of course it depends on how public you want to be with your exception reporting.
 
+## Logging
+Hubot uses [log](https://www.npmjs.com/package/log) to manage logging.
```

Comment:
Add something along the lines of "By default, logging done to stdout, but you can redirect to a file using XY options". Basically, make it clear this is optional.